packages/apputils/package.json

```diff
@@ -45,6 +45,7 @@
     "@phosphor/virtualdom": "^1.1.2",
     "@phosphor/widgets": "^1.6.0",
     "@types/react": "~16.8.8",
+    "printd": "1.3.0",
```

Comment:
Looks like there has been a fix for the firefox issue upstream

Reply:
Yep, just pushed to remove workaround and upgrade the version.